network-libp2p/src/discovery/peer_contacts.rs

```diff
@@ -402,17 +286,17 @@ impl PeerContactInfo {
 
 #[derive(Debug)]
 pub struct PeerContactBook {
-    config: PeerContactBookConfig,
-
     own_peer_contact: PeerContactInfo,
 
     peer_contacts: HashMap<PeerId, Arc<PeerContactInfo>>,
 }
 
 impl PeerContactBook {
-    pub fn new(config: PeerContactBookConfig, own_peer_contact: SignedPeerContact) -> Self {
+    /// If a peer's age exceeds this value in seconds, it is removed (30 minutes)
+    pub const MAX_PEER_AGE: u64 = 1800;
```

Comment:
```suggestion
    pub const MAX_PEER_AGE: u64 = 30 * 60;
```
I'd prefer this to see easily this is 30 min in fact